.pre-commit-config.yaml

```diff
@@ -24,13 +24,35 @@ repos:
     -   id: flake8
         exclude: '^timemachine/cpp'
 
+-   repo: https://github.com/pre-commit/mirrors-mypy
+    rev: v0.942
+    hooks:
+    -   id: mypy
+        additional_dependencies:
+          - "numpy==1.21.4"
```

Comment:
Note: required for [numpy plugin](https://github.com/proteneer/timemachine/pull/711/files#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711R19)